yt_dlp/extractor/archiveorg.py

```diff
@@ -49,6 +49,11 @@ class ArchiveOrgIE(InfoExtractor):
             'upload_date': '20100315',
             'creator': 'SRI International',
             'uploader': 'laura@archive.org',
+            'thumbnail': r're:https://archive\.org/download/.*\.jpg',
+            'release_year': 1968,
+            'display_id': 'XD300-23_68HighlightsAResearchCntAugHumanIntellect.cdr',
+            'track': 'XD300-23 68HighlightsAResearchCntAugHumanIntellect',
```

Comment:
Not strictly related to this PR. But I don't think track makes sense for a lot of these tests

Reply:
track is being added by the extractor, so it should be valid to have it in the test no?

Reply:
The point is that (I feel) track should not be added by the extractor. Or do you think track makes sense here?

Reply:
:confused: 
From what I can see its being added because the file is sometimes a music file with related metadata. 

Are you suggesting we only have `track` if the type is sound?

This seems to have been added a while back, so not sure what their thinking was

Reply:
Yes. Technically, this metadata is wrong. Granted, it is a bug that probably nobody cares about, and in practice, don't really need fixing. But the tests shouldn't imply that it is expected metadata. Just let the tests fail